upgrades/upgradevalidation/migrate.go

```diff
@@ -4,28 +4,20 @@
 package upgradevalidation
 
 import (
-	"github.com/juju/version/v2"
-
 	environscloudspec "github.com/juju/juju/environs/cloudspec"
 )
 
-// ValidatorsForModelMigrationSource returns a list of validators to run source controller for model migration,
+// ValidatorsForModelMigrationSource returns a list of validators to run source
+// controller for model migration.
+// Note: the target version can never be lower than the current version.
 func ValidatorsForModelMigrationSource(
-	targetVersion version.Number, cloudspec environscloudspec.CloudSpec,
+	cloudspec environscloudspec.CloudSpec,
 ) []Validator {
-	validators := []Validator{
-		getCheckTargetVersionForModel(targetVersion, MigrateToAllowed),
```

Comment:
This line is the real meat of it. We no longer check the source for the target version. This is totally up to the target to check for.

Reply:
Can `MigrateToAllowed` be deleted?